docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,213 @@
+    <tunneling-enabled>true</tunneling-enabled>
+    <outbound-enabled>false</outbound-enabled>
+    <enabled>true</enabled>
+    <two-way-ssl-enabled>false</two-way-ssl-enabled>
+    <client-certificate-enforced>false</client-certificate-enforced>
+  </network-access-point>
+</server-template>
+<cluster>
+  <name>cluster-1</name>
+  <cluster-messaging-mode>unicast</cluster-messaging-mode>
+  <dynamic-servers>
+    <name>cluster-1</name>
+    <server-template>cluster-1-template</server-template>
+    <maximum-dynamic-server-count>5</maximum-dynamic-server-count>
+    <calculated-listen-ports>false</calculated-listen-ports>
+    <server-name-prefix>managed-server</server-name-prefix>
+    <dynamic-cluster-size>5</dynamic-cluster-size>
+    <max-dynamic-cluster-size>5</max-dynamic-cluster-size>
+  </dynamic-servers>
+</cluster>
+```
+
+And here is a snippet of off-line WLST that corresponds to the above config.xml snippet:
+
+```
+  templateName = "cluster-1-template"
+  cd('/ServerTemplates/%s' % templateName)
+  templateChannelName = "MyChannel"
+  create(templateChannelName, 'NetworkAccessPoint')
+  cd('NetworkAccessPoints/%s' % templateChannelName)
+  set('Protocol', 't3')
+  set('ListenPort', 7999)
+  set('PublicPort', 30999)
+  set('HttpEnabledForThisProtocol', true)
+  set('TunnelingEnabled', true)
+  set('OutboundEnabled', false)
+  set('Enabled', true)
+  set('TwoWaySslEnabled', false)
+  set('ClientCertificateEnforced', false)
+```
+
+In this example:
+
+- WebLogic binds the custom network channel to port 7999 and the default network channel to 8001.
+
+- The operator will automatically create a Kubernetes service named `DOMAIN_UID-cluster-cluster-1` for both the custom and default channel
+
+- Internal clients running in the same Kubernetes cluster as the channel can access the cluster using `t3://DOMAIN_UID-cluster-cluster-1:8001`.
+
+- External clients would be expected to access the cluster using the custom channel using URLs like `t3://some.public.address.com:30999` or, if leveraging tunneling, `http://some.public.address.com:30999`.
+
+
+#### WebLogic Custom Channel Notes 
+
+- Channel configuration for a configured cluster requires configuring the same network-access-point on each server. The Operator currently doesn't test or support network channels that have a different configuration on each server in the cluster.
+
+- Additional steps are required for external clients beyond configuring the custom channel - see [Approaches](#approaches).
+
+### Setting up a NodePort
+
+#### Getting Started
+
+A Kubernetes NodePort exposes a port on each machine that hosts the Kubernetes cluster where the port is accessible from outside of a Kubernetes cluster. This port redirects network traffic to pods within the Kubernetes cluster. Setting up a Kubernetes NodePort is one of multiple approaches for giving external WebLogic clients access to JMS or EJBs (see [Approaches](#approaches) for alternatives).  
```

Comment:
remove "of multiple"

Reply:
I don't think you need the back ref to approaches.. that was literally just above this right, they should have just read that :)

Reply:
Done.